terraform/modules/prom-ec2/prometheus/main.tf

```diff
@@ -66,14 +66,15 @@ data "template_file" "user_data_script" {
   template = "${file("${path.module}/cloud.conf")}"
 
   vars {
-    config_bucket       = "${aws_s3_bucket.prometheus_config.id}"
-    region              = "${var.region}"
-    targets_bucket      = "${aws_s3_bucket.prometheus_targets.id}"
-    alerts_bucket       = "${aws_s3_bucket.prometheus_config.id}"
-    prom_external_url   = "https://${var.prometheus_public_fqdns[count.index]}"
-    logstash_host       = "${var.logstash_host}"
-    prometheus_htpasswd = "${var.prometheus_htpasswd}"
-    allowed_cidrs       = "${join("\n        ",formatlist("allow %s;", var.allowed_cidrs))}"
+    config_bucket         = "${aws_s3_bucket.prometheus_config.id}"
+    region                = "${var.region}"
+    targets_bucket        = "${aws_s3_bucket.prometheus_targets.id}"
```

Comment:
I'd like to rename this to `ireland_targets_bucket` at some point but not needed now